parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -273,6 +273,21 @@ if [[ ${ID} != "mariner" ]]; then
     /usr/bin/mandb && echo "man-db finished updates at $(date)" &
 fi
 
+# handle starting the logrotate-aks service responsible for log rotation
+if [ -f /etc/systemd/system/logrotate-aks.timer ] && [ -f /etc/systemd/system/logrotate-aks.service ]; then
```

Comment:
remove this, please do it in VHD only

Reply:
you're saying we can and should start the timer in the VHD itself?